.credo.exs

```diff
@@ -1,130 +1,79 @@
+      {Credo.Check.Consistency.SpaceAroundOperators},
+      {Credo.Check.Consistency.SpaceInParentheses},
+      {Credo.Check.Consistency.TabsOrSpaces},
 
-        # For others you can set parameters
+      {Credo.Check.Design.AliasUsage, false},
+      {Credo.Check.Design.DuplicatedCode, excluded_macros: []},
 
-        # If you don't want the `setup` and `test` macro calls in ExUnit tests
-        # or the `schema` macro in Ecto schemas to trigger DuplicatedCode, just
-        # set the `excluded_macros` parameter to `[:schema, :setup, :test]`.
-        {Credo.Check.Design.DuplicatedCode, excluded_macros: []},
+      {Credo.Check.Design.TagTODO},
+      {Credo.Check.Design.TagFIXME},
 
-        # You can also customize the exit_status of each check.
-        # If you don't want TODO comments to cause `mix credo` to fail, just
-        # set this value to 0 (zero).
-        {Credo.Check.Design.TagTODO, exit_status: 2},
-        {Credo.Check.Design.TagFIXME},
+      {Credo.Check.Readability.FunctionNames},
+      {Credo.Check.Readability.LargeNumbers, false},
+      {Credo.Check.Readability.MaxLineLength, false},
+      {Credo.Check.Readability.ModuleAttributeNames},
+      {Credo.Check.Readability.ModuleDoc},
+      {Credo.Check.Readability.ModuleNames},
+      {Credo.Check.Readability.ParenthesesOnZeroArityDefs, false},
+      {Credo.Check.Readability.ParenthesesInCondition},
+      {Credo.Check.Readability.PredicateFunctionNames},
+      {Credo.Check.Readability.SinglePipe, false}, # Off for now
+      {Credo.Check.Readability.Specs, false},
+      {Credo.Check.Readability.StringSigils},
+      {Credo.Check.Readability.TrailingBlankLine},
+      {Credo.Check.Readability.TrailingWhiteSpace},
+      {Credo.Check.Readability.VariableNames},
+      {Credo.Check.Readability.RedundantBlankLines},
 
-        {Credo.Check.Readability.FunctionNames},
-        {Credo.Check.Readability.LargeNumbers},
-        {Credo.Check.Readability.MaxLineLength, priority: :low, max_length: 80},
-        {Credo.Check.Readability.ModuleAttributeNames},
-        {Credo.Check.Readability.ModuleDoc},
-        {Credo.Check.Readability.ModuleNames},
-        {Credo.Check.Readability.ParenthesesOnZeroArityDefs, false},
-        {Credo.Check.Readability.ParenthesesInCondition},
-        {Credo.Check.Readability.PredicateFunctionNames},
-        {Credo.Check.Readability.PreferImplicitTry},
-        {Credo.Check.Readability.RedundantBlankLines},
-        {Credo.Check.Readability.StringSigils},
-        {Credo.Check.Readability.TrailingBlankLine},
-        {Credo.Check.Readability.TrailingWhiteSpace},
-        {Credo.Check.Readability.VariableNames},
-        {Credo.Check.Readability.Semicolons},
-        {Credo.Check.Readability.SpaceAfterCommas},
+      {Credo.Check.Refactor.ABCSize, false},
+      {Credo.Check.Refactor.CondStatements},
+      {Credo.Check.Refactor.DoubleBooleanNegation, false},
+      {Credo.Check.Refactor.FunctionArity, max_arity: 8},
```

Comment:
8 is still many arguments, can we reduce this to 6? I think Ecto must have a much bigger number 😊 .